src/modules/dream/test/DreamBaseTest.cpp

```diff
@@ -66,16 +70,17 @@ std::vector<uint8_t> TestPacket2{
     0x2e, 0x00, 0x00, 0x00, // 0x002e - 46 bytes
     0x11, 0x00, 0x00, 0x00, // Pulse time High (17s)
     0x00, 0x01, 0x00, 0x00, // Pulse time Low (256 clocks)
-    0x11, 0x00, 0x00, 0x00, 0x00, 0x01, 0x00, 0x00, 0x01, 0x00, 0x00,
-    0x00, // Seq number 1
+    0x11, 0x00, 0x00, 0x00,
+    0x00, 0x01, 0x00, 0x00,
+    0x01, 0x00, 0x00, 0x00, // Seq number 1
 
     // Data Header 1
     0x00, 0x01, 0x10, 0x00, // ring 0, fen 1, data size 16 bytes
 
     // Readout 1
     0x22, 0x00, 0x00, 0x00, // tof 34 (0x22)
     0x00, 0x00, 0x14, 0x05, // unused 00 00 module 20, sumo 5
-    0x0e, 0x0b, 0x01, 0x01  // strip 14, wire 11, segment 1, counter 2
+    0x00, 0x00, 0xCC, 0xAA  // strip 14, wire 11, segment 1, counter 2
```

Comment:
The hex values changed but the values in the comment didn't, are they out of date now?